key.ui/src/main/java/de/uka/ilkd/key/gui/prooftree/ProofTreeView.java

```diff
@@ -53,10 +54,21 @@ public class ProofTreeView extends JPanel implements TabPanel {
         ColorSettings.define("[proofTree]gray", "", Color.DARK_GRAY);
     public static final ColorSettings.ColorProperty LIGHT_BLUE_COLOR =
         ColorSettings.define("[proofTree]lightBlue", "", new Color(230, 254, 255));
+    /**
+     * Other green color, currently used for cached goals.
+     */
+    public static final ColorSettings.ColorProperty OTHER_GREEN_COLOR =
```

Comment:
I think we should use the same green color. The difference is currently barely noticeable and brings no benefit in my opinion.